src/main/java/edu/project1/WordsGuesser.java

```diff
@@ -0,0 +1,49 @@
+package edu.project1;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class WordsGuesser {
+    private String word;
+    private String guessedWord;
+    Logger logger = LogManager.getLogger();
+
+    public void setWord(String word) {
+        this.word = word;
+        StringBuilder stringBuilder = new StringBuilder();
+        while (word.length() != stringBuilder.length()) {
+            stringBuilder.append('*');
+        }
+        guessedWord = stringBuilder.toString();
+    }
+
+    public String getWord() {
+        return word;
+    }
+
+    public void getCurrentState() {
```

Comment:
`BUG` плохое название, мб 
```suggestion
    public void printCurrentState() {
```